src/org/labkey/test/tests/AbstractExportTest.java

```diff
@@ -483,7 +492,7 @@ protected BrowserType bestBrowser()
         return BrowserType.CHROME;
     }
 
-    public ColumnHeaderType[] getExportHeaderTypes()
+    protected ColumnHeaderType[] getExportHeaderTypes()
```

Comment:
Just curious why this is changed.

Reply:
I think I was just making `getExportHeaderTypes` and `goToDataRegionPage` consistent.